README.md

```diff
@@ -1,28 +1,142 @@
 # IOS Vast Client
 
-IOS Vast Client is a Swift Framework which implements the [VAST 3.0](https://www.iab.com/guidelines/digital-video-ad-serving-template-vast-3-0/) spec.
+IOS Vast Client is a Swift Framework which implements the [VAST 4.0](https://www.iab.com/guidelines/digital-video-ad-serving-template-vast/) spec.
 
 This project is ongoing and in very early stages of development. As this project is being developed we will only target a subset of the VAST spec to meet our immediate needs. We do not recommend to use this library at this time. 
 
 ## Features
 
-* VAST 3.0 Spec Complient
+* VAST 4.0 Spec Complient
```

Comment:
I would also mention that this is also Vast 3 Spec Compliant or backwards compatible with 3. Same in the description.